### DISCUSSION TITLE
[1252330]死亡循环 本体

### DISCUSSION THREAD

**Discussion by CMDXFFF** [pjy612/SteamManifestCache#92] [Share]
应用ID:1252330

depots
"1254584"
{
"DecryptionKey""cbf97b8608a9ac634d01aeace9b029ba74ee7fa015499a1cb2c4210822fb4983"
}

联系方式:cmdxff@foxmail.com
内测码:9E96218DE4A77642
[12.txt](https://github.com/pjy612/SteamManifestCache/files/13714964/12.txt)

**Comment by pjy612** [comment]
查收
